modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Advantages and disadvantages of non-versioned and versioned cluster tasks
+
```

Comment:
Hi @sounix000. Please add a sentence or short introductory paragraph to each of these files. 

From the OpenShift peer review checklist: "Each assembly and module must contain a sentence or short introductory paragraph after the H1 heading. This serves as the abstract for the file."